### PR TITLE
Feature: Add tasks for Linux and Windows to create custom Python checks

### LINKED ISSUE #315: Support for Custom Check scripts
We are able to add custom_check yamls when deploying but we need the python scripts for them.
We would like the option to copy .py files with the same name as the script to checks.d folder.
It is essential that the script files will be copied with the same process because Datadog requires a restart.

### PR DESCRIPTION
**Pull request by snorlaX-sleeps**
The files for these checks must be included in the parent playbook as the path must be relative to the task, or else an absolute path on the local file-system.
Requires Agent v6+

Fixes: #315

1. Let me know if tests need created / updated for checking this - it has been tested on machines I have been using
2. Let me know if you would prefer it done another way

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,25 @@ To configure a custom check use the configuration below. This creates the corres
           - some_data: true
 ```
 
+##### Custom Python Checks
+
+To configure a custom check and to pass a Python check to the Playbook, please use the configuration below.  
+This requires the Datadog play / role to be part of a larger Playbook, with the value passed in being the relative filepath 
+to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 
+This is only available for Datadog v6+.
```
Comment (alai97) [inline README.md]: Would it be helpful to add a reference link for Play and Role (https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#playbook-keywords)?

```suggestion
To pass a Python check to the playbook, use the configuration below. 

This configuration requires the Datadog play and role to be a part of the larger playbook where the value passed in is the relative file path to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 

This is only available for Agent v6+.  
```

File: README.md [hunk 2]
```diff
@@ -110,6 +110,25 @@ To configure a custom check use the configuration below. This creates the corres
...
 ```
 
+##### Custom Python Checks
+
+To configure a custom check and to pass a Python check to the Playbook, please use the configuration below.  
+This requires the Datadog play / role to be part of a larger Playbook, with the value passed in being the relative filepath 
+to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 
+This is only available for Datadog v6+.  
+
+The key should be the name of the file created in the checks directory `checks.d/{{ item }}.py`
```
Comment (alai97) [inline README.md]:
```suggestion
The key should be the name of the file created in the checks directory `checks.d/{{ item }}.py`:
```

File: tasks/agent-win.yml [hunk 3]
```diff
@@ -51,6 +51,13 @@
   with_items: "{{ datadog_checks|list }}"
   notify: restart datadog-agent-win
 
+- name: Create custom check file for each Custom check
```
Comment (alai97) [inline tasks/agent-win.yml]:
```suggestion
- name: Create custom check file for each custom check
```

File: tasks/agent-linux.yml [hunk 4]
```diff
@@ -84,6 +84,16 @@
   with_items: "{{ datadog_checks|list }}"
   notify: restart datadog-agent
 
+- name: Create custom check file for each Custom check
```
Comment (alai97) [inline tasks/agent-linux.yml]:
```suggestion
- name: Create custom check file for each custom check
```

**Review by alai97 — APPROVED** [review]
Looks good pending some copy nits!

**Comment by bkabrda** [comment]
Hi @snorlaX-sleeps, thanks a lot for submitting this PR. I did some testing locally and it looks like everything works fine. As soon as you take a look at the docs suggestions by @alai97 and accept/comment on them, I can get this merged.

**Comment by snorlaX-sleeps** [comment]
@alai97 - PR updated with the suggestions

**Review by bkabrda — APPROVED** [review]
Works and looks fine, I'm going to merge. Thanks a lot for the PR!

**Pull request merged**